messages/backup_commands.options

```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-CheckBackupResponse.id fixed_length:true max_size:256
-ListBackupsResponse.info max_count:100
+CheckBackupResponse.id fixed_length:true max_size:65
```

Comment:
+1 byte to account for `'\0'`, right?

Reply:
yeah. keep in mind `.options` is only for nanopb, which uses null terminated strings for `string`.